aws_lambda/sample/sync_though_cloud_storage/go/main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
```

Comment:
I think customer will require run `go get github.com/aws/aws-sdk-go/service/s3/s3manager` if yes, then we need to document this also README.md